### PR TITLE
Remove redundant binary specification

### PR DESCRIPTION
**Pull request by athei**
The binary of our template node was named `parachain-collator` and the name of the actual node `polkadot-parachain`. This proofed to be confusing for people which mistakenly thought that `parachain-collator` would be "the real thing". Evidence of this is that the template was deployed for the contracts node making the contracts RPCs unavailable.

This PR renames the `parachain-collator` binary to `parachain-template-node` and removes the redundant explicit specification of the binary.

### CODE REVIEW

**Review by bkchr — APPROVED** [review]

**Comment by gilescope** [comment]
Excellent <3

**Review by chevdor — APPROVED** [review]

File: parachain-template/polkadot-launch/config.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
 	},
 	"parachains": [
 		{
-			"bin": "../target/release/parachain-collator",
+			"bin": "../target/release/polkadot-parachain",
```
Comment (gilescope) [inline parachain-template/polkadot-launch/config.json:20]: as this is part of the template, should the bin being executed be `polkadot-template-node` ?

**Review by gilescope — APPROVED** [review]

File: parachain-template/polkadot-launch/config.json (hunk 1 above)
Comment (gilescope) [inline parachain-template/polkadot-launch/config.json:20]:
```suggestion
			"bin": "../target/release/polkadot-template-node",
```

Comment (bernardoaraujor) [reply parachain-template/polkadot-launch/config.json:20]: `parachain-template-node`

Comment (athei) [reply parachain-template/polkadot-launch/config.json:20]: I think the rename as I did it is what we want. It was using the template but it was a mistake IMHO.

**Comment by joepetrowski** [comment]
bot merge

**Pull request merged**